src/Microsoft.AspNetCore.WebSockets/WebSocketMiddleware.cs

```diff
@@ -119,10 +119,7 @@ public bool IsWebSocketRequest
 
                 Stream opaqueTransport = await _upgradeFeature.UpgradeAsync(); // Sets status code to 101
 
-                // Allocate a buffer for receive (default is 4k)
-                var buffer = new byte[receiveBufferSize];
-
-                return WebSocketProtocol.CreateFromStream(opaqueTransport, isServer: true, subProtocol: subProtocol, keepAliveInterval: keepAliveInterval, buffer: buffer);
+                return WebSocketProtocol.CreateFromStream(opaqueTransport, isServer: true, subProtocol: subProtocol, keepAliveInterval: keepAliveInterval);
```

Comment:
What about logging a warning if the user provided a buffer? Something like "Ignoring Receive Buffer, it is no longer supported"

Reply:
It was never supported in a released version anyways
https://github.com/aspnet/WebSockets/commit/6de76c5e69bd74b8d5742642e1e90a2a39874323#diff-c5fcb32825a2f3acc07e050ba5f20502

Reply:
We set the default to 4K, so it's currently always set. We could change that but the risk is so low, it's not worth it. When somebody complains that the websocket isn't allocating the 4K they asked it to, we can "fix it".

Reply:
As @BrennanConroy pointed out, I think we're good if we never honored it before :)

Reply:
It was in 2.0